doc/source/userguide.rst

```diff
@@ -192,70 +208,79 @@ Currently, two backends exist:
   improvement is currently small in 1D and 3D, but work is underway to further
   optimize this backend.
 
+This backend can be specified via::
+
+    import torch
+    from kymatio import Scattering2D
+    from kymatio.scattering2d.backend.torch_skcuda_backend import backend
+
+    scattering = Scattering2D(J=2, shape=(32, 32), frontend='torch', backend=backend)
+
+
 Benchmarks
```

Comment:
TODO: Rerun these timings.

Reply:
They shouldn't have changed (except we could add tf and np backend but i'm not sure how it works)

Reply:
I've open an issue.